src/cloudai/workloads/nccl_test/slurm_command_gen_strategy.py

```
@@ -31,19 +31,29 @@ def image_path(self) -> str | None:
         tdef: NCCLTestDefinition = cast(NCCLTestDefinition, self.test_run.test)
         return str(tdef.docker_image.installed_path)
 
+    def gen_srun_prefix(self, use_pretest_extras: bool = False, with_num_nodes: bool = True) -> List[str]:
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)